etcdutl/etcdutl/snapshot_command.go

```diff
@@ -38,6 +38,7 @@ var (
 	restorePeerURLs     string
 	restoreName         string
 	skipHashCheck       bool
+	initialMmapSize     = uint64(10 * 1024 * 1024 * 1024)
```

Comment:
It might be better to reuse the existing `initialMmapSize` defined in server/storage/backend? We need to export that variable.

Reply:
Hey @ahrtr, should I proceed to make this change? Or do you think I should wait for consensus from more maintainers?

Reply:
Please feel free to make the change. We will request maintainers to review & approve later.

Reply:
Done.